pointers_examples.cpp

```diff
@@ -0,0 +1,35 @@
+/*#include<iostream>
+int main()
+{
+	int a = 65;                            //by not providing ascii values in this it will genrate garbage values 
+	void* ptr = &a;
+	char* ptr1 = (char*)&a;                //this will hold only the character values depending on 'a'
+ 	std::cout << *ptr1 << std::endl;       //this will give the value of 'a'
+	std::cout << *(ptr1 - 1) << std::endl; //this decreases the initial position of pointer 
+	std::cout << *(ptr1 + 1) << std::endl; //this increases the initial position of pointer
+	std::cout << *(ptr1) << std::endl;     //this will also gives the value of 'a'
+	return 0;
+}*/
+
+//-------------------------------------------------------------------------------------------------------------------------------------------------

```

Comment:
why there is unwanted line here,you should not do this.
It might give error while compiling.